core/src/main/resources/error/error-classes.json

```
@@ -1414,6 +1414,11 @@
       "sortBy must be used together with bucketBy."
     ]
   },
+  "SQL_CONF_NOT_FOUND" : {
+    "message" : [
+      "The SQL config \"<key>\" cannot be found. Please verify that the config exists."
```

Please, quote the config in the code and remove "" around the key:
```suggestion
      "The SQL config <sqlConf> cannot be found. Please verify that the config exists."
```